### ISSUE TITLE
crash on auto-dewarp

### ISSUE THREAD

**Issue by graytape** [4lex4/scantailor-advanced#184]
Hi,
one time out of two I try to batch auto-dewarp on output the program crashes, and if I try to reopen without deactivating the auto dewarp output it crashes again.
Is there a log file somewhere?

It happens also reducing the load of work on the CPU

Thanks

**Comment by bui89** [comment]
Yes, it's very annoying issue. it crashes randomly and no log to find out what is the problem.

**Comment by mara004** [comment]
You could try invoking STA from the command-line and take a look at the output, or launch the program in gdb debugger and create a backtrace when the crash occurs.

**Comment by kugland** [comment]
With me it crashes especially in pages where, after deskewing, part of the content box is outside of the image. 

what():  fillMarginsInPlace: the content area exceeds image rect.

![image](https://github.com/4lex4/scantailor-advanced/assets/1173932/dab0094d-119a-4576-b90a-d9a7fbf24a91)

Changing the content rect or disabling auto-dewarp makes the crash go away.

**Comment by mvasi90** [comment]
Same issue over and over again. It is exhausting to use ScanTailor.

**Comment by cobsvilla** [comment]
Same issue here.
But in "Output" tab I've tried to change "Mode" from "Black and White" to "Color/Grayscale" and, so far, it seems to fix the problem

**Comment by Sissel-Ng** [comment]
Still have not been fixed unfortunately.
